### PR TITLE
Remove "sendall"

### PR DESCRIPTION
**Pull request by DUFCLiam**
#15

### CODE REVIEW

**Comment by rsajeey** [comment]
There's no point on having this PR, instead another PR was created before you https://github.com/TotalFreedom/TF-Essentials/pull/13 and this is considered as Duplicate of https://github.com/TotalFreedom/TF-Essentials/pull/13

**Comment by DUFCLiam** [comment]
I seen sorry for the duplicate

{CLOSED}

**Comment by JeromSar** [comment]
@rsajeey @DUFCLiam No, that is not true. PRs are never duplicates.

**Comment by DUFCLiam** [comment]
According to what @JeromSar said
 
{REOPNED}

File: Essentials/src/com/earth2me/essentials/commands/Commandmail.java [hunk 1]
```diff
@@ -122,15 +110,6 @@ protected void run(Server server, CommandSource sender, String commandLabel, Str
...
-    private class SendAll implements Runnable {
-
-        String message;
-
-        public SendAll(String message) {
-            this.message = message;
-        }
-
         @Override
         public void run() {
```
Comment (rsajeey) [inline Essentials/src/com/earth2me/essentials/commands/Commandmail.java]: This is un-used.

File: Essentials/src/com/earth2me/essentials/commands/Commandmail.java [hunk 2]
```diff
@@ -122,15 +110,6 @@ protected void run(Server server, CommandSource sender, String commandLabel, Str
...
-
-    private class SendAll implements Runnable {
-
-        String message;
-
-        public SendAll(String message) {
-            this.message = message;
-        }
-
         @Override
```
Comment (rsajeey) [inline Essentials/src/com/earth2me/essentials/commands/Commandmail.java]: 113 - 122 is un-needed.

**Pull request closed without merging**